src/app/components/transactionsRequests/ContractCallRequest.tsx

```diff
@@ -14,12 +14,15 @@ import {
   Coin,
   ContractFunction,
   extractFromPayload,
+  isMultiSig,
 } from '@secretkeylabs/xverse-core';
+import { buf2hex } from '@secretkeylabs/xverse-core/utils/arrayBuffers';
```

Comment:
```suggestion
  isMultiSig,
  buf2hex,
} from '@secretkeylabs/xverse-core';
```

Reply:
for more context, there's an effort to remove imports of core code which was not built by esm

see this PR: https://github.com/secretkeylabs/xverse-core/pull/235